var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -77,6 +83,10 @@ class Tasmanian(CMakePackage):
     conflicts('+mpi', when='@:5.1')    # MPI is broken prior to 6.0
     conflicts('+xsdkflags', when='@:5.1')  # 6.0 is the first version included in xSDK
 
+    # patching some bugs
+    patch('addons70.patch', when='@7.0', sha256='f9e0a08f3f6ce916c8133096996acb7d5df699d12ed540707c8b1bc4481e5a53')
+    patch('packageconf70.patch', when='@7.0', sha256='e4ccfb63933a684e7cba7783e09188af30023597c95dd2374fb3e882add5b205')
```

Comment:
These checksums aren't required since the patch is already in Spack and doesn't need to be downloaded.